### ISSUE TITLE
The learning curve of the FewShot step

### ISSUE THREAD

**Issue by semi-supervised-paper** [phecy/SSL-FEW-SHOT#5]
Hi,
Thanks for sharing your code.  And I wonder if you can share the curves (like training loss, training acc, val loss, val acc), since my training curves seem to be strange.

**Comment by phecy** [comment]
```
python train_protonet.py --lr 0.0001 --temperature 128   \
--max_epoch 100 --model_type AmdimNet --dataset MiniImageNet \
--init_weights ./saves/initialization/miniimagenet/mini_imagenet_ndf192_rkhs1536_rd8_ssl_cpt.pth  \
--save_path ./MINI_ProtoNet_MINI_1shot_5way/ \
--shot 1  --way 5 --gpu 4 --step_size 10 --gamma 0.5 \
--ndf 192 --rkhs 1536 --nd 8
```
the log of the above training  command  have been upload to the github code

**Comment by semi-supervised-paper** [comment]
Thanks for your response.
